storey/steps/assertion.py

```diff
@@ -0,0 +1,149 @@
+from dataclasses import dataclass
+from typing import Callable, List, Any, Collection
+
+from storey.dtypes import _termination_obj
+from storey.flow import Flow
+
+
+@dataclass
+class _Operator:
+    str: str
+    fn: Callable[[Any, Any], bool]
+
+    def __call__(self, x, y):
+        return self.fn(x, y)
+
+    def __str__(self):
+        return self.str
+
+
+_EQUALS = _Operator("==", lambda x, y: x == y)
+_NOT_EQUALS = _Operator("!=", lambda x, y: x != y)
+_ABOVE = _Operator(">", lambda x, y: x > y)
+_BELOW = _Operator("<", lambda x, y: x < y)
+_ABOVE_OR_EQUAL = _Operator(">=", lambda x, y: x >= y)
+_BELOW_OR_EQUAL = _Operator("<=", lambda x, y: x <= y)
+
```

Comment:
_GREATER_THAN
_LESS_THAN
?
Never heard of these operators being refered to as above and below.